S/SCS_GPU/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+using BinaryBuilder
+
+name = "SCS"
+version = v"2.1.1"
+
+# Collection of sources required to build SCSBuilder
+sources = [
+    GitSource("https://github.com/cvxgrp/scs.git", "e6ab81db115bb37502de0a9917041a0bc2ded313")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/scs*
+flags="DLONG=0 USE_OPENMP=0 BLAS64=1 BLASSUFFIX=_64_"
+blasldflags="-L${prefix}/lib -lopenblas64_"
```

Comment:
Libraries are in `${libdir}`, see https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables, not necessarily in `${prefix}/bin`, better to use the most specific variable
```suggestion
blasldflags="-L${libdir} -lopenblas64_"
```